bims/static/js/views/olmap.js

```diff
@@ -350,6 +350,19 @@ define([
             this.bioLegendView = new BioLegendView();
             this.$el.append(this.bioLegendView.render().$el);
 
+            $('#map-legend-wrapper').draggable({
```

Comment:
```suggestion
            var $mapLegendWrapper = $('#map-legend-wrapper');
            $mapLegendWrapper.draggable({
```

And also update all of the $('#map-legend-wrapper') to use this variable.

Reply:
@ann26

Reply:
hi @dimasciput I'm not sure why it is necessary to use a variable for that?

Reply:
@ann26 , It is a bad practice that I also try to avoid it as much as I can : 

1. $() creates a new jQuery object, in this case you keep creating new object
2. It is easier if you want to change id or class name, you don't have to change every selector, this can also be applied to all programming languages

Reply:
hi @ann26 
what dimas said that when we define $('#map-legend-wrapper') to $mapLegendWrapper, it is basically put 1 slot in memory for $mapLegendWrapper to pointing to $('#map-legend-wrapper')

if we don't define it, it will create 1 slot for everytime we call $('#map-legend-wrapper'), it will be bad for memory management

Yes, i'm also usually forget it and trying to avoid this also

Reply:
@ann26 can you also updates your code here : https://github.com/kartoza/django-bims/pull/481/files#diff-1553556d2930aabbef13964b5f2115fcR272

Try pull from the develop branch first, I made some changes in the app.js

Reply:
PS :
You can always rely on pycharm to check the duplicates : 
![image](https://user-images.githubusercontent.com/1979569/48242852-b0674380-e40f-11e8-844f-a14883454856.png)

Reply:
hi @meomancer and @dimasciput , thanks for the explanation, I have changed it